front/src/applications/operationalStudies/components/Scenario/ScenarioDescription.tsx

```diff
@@ -25,14 +23,30 @@ const ScenarioDescription = ({
   const { openModal } = useModal();
 
   return (
-    <div className="scenario-details">
+    <div>
       <div className="scenario-details-name">
         <span className="flex-grow-1 scenario-name text-truncate" title={scenario.name}>
           {scenario.name}
         </span>
+      </div>
+
+      <div className="scenario-description">
+        {scenario.description && (
+          <div className="scenario-details-description">{scenario.description}</div>
+        )}
+        <div className="flex justify-end">
+          <button
+            type="button"
+            className="scenario-collapse-button"
+            aria-label={t('toggleTimetable')}
+            onClick={() => collapseTimetable()}
+          >
+            <ChevronLeft />
+          </button>
+        </div>
         <button
```

Comment:
Seems like the update button is always aligned with the electrical_profile/infra block, why not put it inside ?

Reply:
No, its top edge is 16px from the baseline of the last line of the description.

Reply:
Ok not what I meant but it's not that important. But we could leave a comment next to the `bottom: -3rem;` in the css :

```suggestion
        bottom: -3rem; // button has to be positioned regarding the description
```

Reply:
Done